### DISCUSSION TITLE
Commercial　Application

### DISCUSSION THREAD

**Discussion by ccc-111** [BaranziniLab/KG_RAG#12] [Q&A] [answered]
Hello, can I apply your project ideas to commercial projects?

**Comment by karthiksoman** [comment, marked as answer]
Hello!
Thank you for your inquiry and your interest in our project ideas!
Regarding the application of these ideas to commercial projects, UCSF has already assigned all the intellectual property rights related to the commercial use of projects involving SPOKE, including KG-RAG, to a company named 'Mate'. You are encouraged to explore more about 'Mate' from [here](https://www.matebioservices.com/). If you have any additional questions or need further clarification, please don't hesitate to ask!
